tests/openvino/test_modeling.py

```diff
@@ -1134,6 +1134,10 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "decilm",
     )
 
+    SUPPORTED_SSM_ARCHITECTURES = ("mamba",)
+    if is_transformers_version(">=", "4.39"):
+        SUPPORTED_ARCHITECTURES += SUPPORTED_SSM_ARCHITECTURES
+
```

Comment:
question : are we testing all cases in addition of stateful mode, if not would you mind adding a test for this as well to make sure all different cases are supported ? other possibility : could also make sense to only support stateful mode for newly added architectures so we could also restrict the export to stateful mode only

Reply:
yes, let us support only stateful mode